integration-test/src/main/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -338,6 +343,9 @@ private void createTables_ShouldCreateTables() throws Exception {
   private void deleteTables_ShouldDeleteTables() throws Exception {
     // Act
     int exitCode = executeWithArgs(getCommandArgsForDeletion(CONFIG_FILE_PATH, SCHEMA_FILE_PATH));
+    if (exitCode != 0 && couldFailToReadNamespaceAfterDeletingTable()) {
```

Comment:
This case could fail and retry is needed with YugabyteDB.

Reply:
Good point. Only this test case requires long wait (e.g., 15 seconds) on my laptop, so I chose the retry.

But, I just found `--ysql_enable_db_catalog_version_mode=false` option for `bin/yugabyted` and it prevented "Catalog Version Mismatch" error with the test cases. I'll use the option for the integration tests and remove unnecessary waits as much as possible.

Reply:
Fixed it with https://github.com/scalar-labs/scalardb/pull/1710/commits/cf633158438e27644c4c5d6d73855d3a0e9ebc3d and https://github.com/scalar-labs/scalardb/pull/1710/commits/f0269b18d410c9de80e0233d0aade382febf7b74.

The option especially works for this case. Some other test cases still need waits, so I left them as is.